install/kubernetes/helm/istio/charts/mixer/templates/config.yaml

```diff
@@ -536,7 +536,7 @@ spec:
       destination_principal: destination.principal | "unknown"
       destination_app: destination.labels["app"] | "unknown"
       destination_version: destination.labels["version"] | "unknown"
-      destination_service: destination.service.host | "unknown"
+      destination_service: destination.service.host | request.host | "unknown"
```

Comment:
for TCP using `request.host` is not correct

Reply:
Yes, destination ip should be used in this case.

Reply:
Can you easily convert `destination.ip` into string type and set it like you suggested. That would be super useful!!

Reply:
I think mixer does not support type conversion expression.. so I guess the work around is to set dashboard differently for TCP metric?

Reply:
yeah, for now it would require you to first enable `destination.ip` as a label in TCP metrics and then setup your dashboard.

Reply:
Oh, I did not realize destination IP is only a label for tcp log but not tcp metrics. I guess there is some concerns about carnality? @douglas-reid